packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/ConfirmBulkActionDialog/index.js

```diff
@@ -104,6 +104,8 @@ const ConfirmDialogPublishAll = ({ isOpen, onToggleDialog, isConfirmButtonLoadin
       return data;
     },
     {
+      // It avoid to call the Query if no IDS are passed
```

Comment:
I can understand this code without the comment, it might be more useful to illustrate when this situation might happen, e.g: "Sometimes by magic the confirm dialog will open even though nothing is selected, it's an edge case but this avoids sending a query with bad data"